go-router/main/default/classes/GoRouterSetup.cls

```diff
@@ -174,6 +174,33 @@ public without sharing class GoRouterSetup extends GoRouter {
     return '';
   }
 
+  /*
+  * @description Used by VF page to retrieve all valid pages that can be routed to
+  * and the inputs that lead to them. Constructs this list one and stores it as
+  * allInputs, which future calls will then return directly.
+  * @return Map of String input to String combined list of valid shortcuts.
+  */
+  public Map<String, String> getAllInputs {
```

Comment:
Shouldn't this just be called `allInputs` since it's a property? Otherwise the `allInputs` check below will always be null.